docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -69,57 +69,53 @@ The following guidelines describe the required structure for your collection:
 
-The collection MUST document all eligible controller Python versions that are not supported in the controller environment. See :ref:`coll_python_docs_req` for details.
+  * controller environment: the plugins/modules always run in the same environment (Python interpreter, venv, host, and so on) as ansible-core itself.
+  * eligible controller Python version: a Python version that is supported on the controller side by at least one ansible-core version that the collection supports. The eligible versions can be determined from the :ref:`ansible_core_support_matrix` and from the ``requires_ansible`` value in ``meta/runtime.yml`` in the collection.
+
+* The collection MUST document all eligible controller Python versions that are **not** supported in the controller environment. See :ref:`coll_python_docs_req` for details.
 
 Other environment
 ~~~~~~~~~~~~~~~~~
 
-Collections MUST support all eligible controller Python versions in the other environment, unless required libraries do not support these Python versions. The :ref:`Steering Committee <steering_responsibilities>` can grant other exceptions on a case-by-case basis. 
+* Collections MUST support all eligible controller Python versions in the other environment, unless required libraries do not support these Python versions. The :ref:`Steering Committee <steering_responsibilities>` can grant other exceptions on a case-by-case basis. 
 
-Collections SHOULD support all eligible target Python versions in the other environment.
+  * other environment: the plugins/modules run not in a controller environment.
+  * eligible target Python version: a Python version that is supported on the target side by at least one ansible-core version that the collection supports. The eligible versions can be determined from the :ref:`ansible_core_support_matrix` and from the ``requires_ansible`` value in ``meta/runtime.yml`` in the collection.
 
-The collection MUST document all eligible target Python versions that are not supported in the other environment. See :ref:`coll_python_docs_req` for details.
+* The collection MUST document all eligible target Python versions that are not supported in the other environment. See :ref:`coll_python_docs_req` for details.
 
-.. note::
+Dropping Python versions support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    Note that dropping support for a Python version for an existing module/plugin is a breaking change, and thus requires a major release. A collection MUST announce dropping support for Python versions in their changelog, if possible in advance (for example, in previous versions before support is dropped).
+Because dropping support for a Python version for an existing module/plugin is a breaking change, the collection:
+
+* SHOULD announce it under the breaking changes section in its changelog in previous versions before the support is dropped.
```

Comment:
The breaking change only happens in the new major release, so this should be `deprecated_features`:
```suggestion
* SHOULD announce it under the deprecated features section in its changelog in previous versions before the support is dropped.
```